src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final String CHARACTER = "w";
```

Comment:
if we don't use variable outside of class it's better to make it private to follow encapsulation principle